src/schema/journey.test.ts

```diff
@@ -816,8 +767,8 @@ async function newClassObject(
       },
       stopwords: {
         preset: 'en',
-        additions: null,
-        removals: null,
+        additions: null as unknown as undefined, // hack to deal with weird typing
+        removals: null as unknown as undefined, // hack to deal with weird typing
```

Comment:
 😭

Reply:
Hahah yeh, it's because the `WeaviateClass` type that comes from the OpenAPI schema does not contain `null` but I want to be able to type this method for ease of use. I will look again at why this is `null` rather than `string[] | undefined` as it is in the `WeaviateClass` type to be sure

Reply:
Okay there's no parsing in the TS client; Weaviate returns a `null` here rather than `undefined` despite the typing. How confusing!